### DISCUSSION TITLE
CAF enterprise-scale landing zone (small and midsize enterprise) TreyResearch

### DISCUSSION THREAD

**Discussion by melwinlawrance** [Azure/Enterprise-Scale#570] [Q&A]
Guys,

Can anybody share with me the Visio file for TreyResearch?

Thanks in advance!

Regards,
Melwin Lawrance

**Comment by FallenHoot** [comment]
A version of this has been provided. Just have to remove the .txt part
[ES_TreyResearch.vsdx.txt](https://github.com/Azure/Enterprise-Scale/files/6549127/ES_TreyResearch.vsdx.txt)

**Reply by melwinlawrance** [reply]
Thanks!
